RandBLAS/sparse.hh

```diff
@@ -136,18 +136,18 @@ SparseSkOp<T>::SparseSkOp(
     }
 };
 
-template <typename T>
-SparseSkOp<T>::~SparseSkOp() {
+template <typename T, typename RNG>
+SparseSkOp<T,RNG>::~SparseSkOp() {
     if (this->own_memory) {
         delete [] this->rows;
         delete [] this->cols;
         delete [] this->vals;
     }
 };
 
-template <typename T>
+template <typename SKOP>
 static bool has_fixed_nnz_per_col(
-    SparseSkOp<T> &S0
+    SKOP const& S0
```

Comment:
Why is this preferable to ``SparseSkOp<T>``?

Reply:
it's simpler to use, fewer template parameter